test/src/com/xilinx/rapidwright/rwroute/TestRouterHelper.java

```diff
@@ -240,35 +242,44 @@ public void testInvertPossibleGndPinsToVccPinsLutInput(boolean invertLutInputs)
         Assertions.assertEquals("O=!I0", LUTTools.getLUTEquation(cell));
 
         Net gndNet = design.getGndNet();
-        gndNet.createPin("A6", cell.getSiteInst());
+        SitePinInst spi = gndNet.connect(cell, "I0");
```

Comment:
This test is actually failing due to this line, but it due to a deficiency in `Net.connect()`.  I will have some improvements to that API soon.